transforms/angle-brackets/__testfixtures__/deeply-nested-sub.output.hbs

```diff
@@ -7,11 +7,20 @@
 
 <SomeComponent class={{concat foo (some-helper bar)}} />
 <SomeComponent class={{concat foo (some-helper bar quuz)}} />
-<SomeComponent @person={{hash name="Sophie" age=1}} />
+<SomeComponent
+  @person={{hash name="Sophie" age=1}}
+  @message={{t "welcome" count=1}}
+ />
```

Comment:
prettier seems to insert a space before `/>`

Reply:
Yep, for self-closing components it seems to indent them slightly.